### PR TITLE
Fix privilege name

### PR DESCRIPTION
**Pull request by AmitPhulera**
## Product Description

While refactoring https://github.com/dimagi/commcare-hq/pull/32049 this occurrence was not changed.
## Safety Assurance
Tested on local
### QA Plan

NA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/smsbillables/management/commands/add_sms_gateway_permissions.py [hunk 1]
```diff
@@ -61,10 +61,10 @@ def handle(self, usernames, **options):
                         from_role=user_role.role,
                         to_role=global_sms_gateway_access,
                     )
-                    print("Enabled privilege to Edit Global SMS Gateways for the user {user.username}")
+                    print(f"Enabled privilege to Edit Global SMS Gateways for the user {username}")
                 else:
                     print("User %s already have the requested privilege"
                           % user.username)
```
Comment (RynhardPietersen) [inline corehq/apps/smsbillables/management/commands/add_sms_gateway_permissions.py:67]: shouldn't this also just be `username` instead of `user.username`?

**Review by RynhardPietersen — APPROVED** [review]
I don't see a problem here. I just have that one question. :)

File: corehq/apps/smsbillables/management/commands/add_sms_gateway_permissions.py (hunk 1 above)
Comment (AmitPhulera) [reply corehq/apps/smsbillables/management/commands/add_sms_gateway_permissions.py:67]: We have a user object with us that has a username attribute. I guess it should work fine.

Comment (AmitPhulera) [reply corehq/apps/smsbillables/management/commands/add_sms_gateway_permissions.py]: For consistency, I have updated it to be `user.username` at both the places https://github.com/dimagi/commcare-hq/pull/32115/commits/7c4d4ee787b1011b103b169b6d4d3b030446daa9

Comment (RynhardPietersen) [reply corehq/apps/smsbillables/management/commands/add_sms_gateway_permissions.py]: that makes sense. Thanks!

**Pull request merged**